lib/src/base-styles/_nx-globals.scss

```diff
@@ -15,8 +15,11 @@ $-nx-scrollable-default-height: 400px;
 .nx-text-link {
   @include nx-container-helpers.container-horizontal;
 
+  border: 1px solid transparent;
   color: var(--nx-color-link);
   cursor: pointer;
+  margin: 0 calc(0px - #{var(--nx-spacing-base)});
+  padding: 0 var(--nx-spacing-base);
```

Comment:
You're margin adjustment should cancel out not only the padding but the border too.  So it's off by one.  Looking at the mockup, the intent appears to be that it be 4px from the content to the _outside_ of the border, so this padding actually needs to be decreased by 1.

Reply:
> You're margin adjustment should cancel out not only the padding but the border too. So it's off by one. Looking at the mockup, the intent appears to be that it be 4px from the content to the _outside_ of the border, so this padding actually needs to be decreased by 1.

Fixed here and confirmed by measuring screenshots: https://github.com/sonatype/sonatype-react-shared-components/pull/502/commits/f306a7ab262b132447388637a981726832f6b0ca

Reply:
> Fixed here and confirmed by measuring screenshots: f306a7a

That has too little margin.  Margin should cancel out padding + border

Reply:
> > Fixed here and confirmed by measuring screenshots: [f306a7a](https://github.com/sonatype/sonatype-react-shared-components/commit/f306a7ab262b132447388637a981726832f6b0ca)
> 
> That has too little margin. Margin should cancel out padding + border

This should fix it: https://github.com/sonatype/sonatype-react-shared-components/pull/502/commits/3d2d12276dd6f1faed2670ba8f494fcb7e139983